modules/edu.gemini.seqexec.server/build.sbt

```diff
@@ -0,0 +1,20 @@
+import Settings._
+import Settings.Libraries._
+import Settings.LibrariesJVM._
+
+name := "edu.gemini.seqexec.server"
+
+libraryDependencies ++= Seq(
+  ScalaZCore,
+  ScalaZConcurrent,
+  "io.argonaut"        %% "argonaut"                  % "6.1",
+  "commons-httpclient" % "commons-httpclient"         % "2.0",
+  // OCS bundles
+  SpModelCore,
+  POT,
+  EpicsACM
+)
+
+unmanagedJars in Compile ++= Seq(
+  new File((baseDirectory in ThisBuild).value, "lib/squants_2.11-0.6.1.jar")
```

Comment:
I think only the latest version is cross built for scala.js, for what it's worth.  We depend on squants in core so if we want to cross build it we'll need the latest version.  I'm also no sure whether Florian's additions have been officially incorporated into squants.